### DISCUSSION TITLE
Cannot connect to REDIS (Error: connect EHOSTUNREACH 192.168.64.2:6379)

### DISCUSSION THREAD

**Discussion by mprajescu** [misskey-dev/misskey#13988] [General]
Hello, 

I followed the guide provided here: https://github.com/misskey-dev/misskey/blob/develop/docs/docker.en.md 
I updated the default.yml db and redis hostnames from "localhost" to "db" and "redis" as per instructions. 

I receive this error when I start the stack. 

```
web_1    | 
web_1    | > misskey@12.91.0 migrateandstart
web_1    | > npm run migrate && npm run start
web_1    | 
web_1    | 
web_1    | > misskey@12.91.0 migrate
web_1    | > ts-node ./node_modules/typeorm/cli.js migration:run
web_1    | 
web_1    | Error: connect EHOSTUNREACH 192.168.64.2:6379
web_1    |     at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1142:16) {
web_1    |   errno: -113,
web_1    |   code: 'EHOSTUNREACH',
web_1    |   syscall: 'connect',
web_1    |   address: '192.168.64.2',
web_1    |   port: 6379
web_1    | }
misskey_web_1 exited with code 1
```

I left elasticsearch commented and disabled. 
Doesn't matter if I choose to build the docker image or I use the image misskey/misskey:latest, the issue still occurs. 

Can you please advise how to fix the connectivity issue or if there is anything else requried for further troubleshooting?

Thanks,
Mihai

**Comment by AMoonRabbit** [comment]
Could you share you default.yml minus passwords and other sensitive data? If it's still an issue.
